recipes/libid3tag/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "libid3tag" %}
+{% set version = "0.16.2" %}
+{% set sha256 = "96198b7c8803bcda44e299615e1929a85bd5a5da02e67ebc442735bc50018190" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tenacityteam/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [osx]
```

```suggestion
  skip: True  # [osx]
  run_exports:
    # TODO: Relax pinning when API/ABI reaches 1.0 
    - {{ pin_subpackage('libid3tag', exact=True) }}
  ignore_run_exports_from:
    - {{ compiler('cxx') }}
```
[Link inspector reports](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=504385&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1265) overdepending on the cxx standard library. Need to either ignore exports from the compiler or remove the compiler if it is not needed.